cpp/models/glct_secir/parameters.h

```diff
@@ -0,0 +1,680 @@
+/* 
+* Copyright (C) 2020-2024 MEmilio
+*
+* Authors: Lena Ploetzke
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#ifndef MIO_GLCT_SECIR_PARAMS_H
+#define MIO_GLCT_SECIR_PARAMS_H
+
+#include "memilio/config.h"
+#include "memilio/utils/parameter_set.h"
+#include "memilio/math/eigen.h"
+#include "memilio/math/floating_point.h"
+#include "memilio/epidemiology/uncertain_matrix.h"
+#include "memilio/utils/logging.h"
+
+namespace mio
+{
+namespace glsecir
+{
+
+/***********************************************
+* Define Parameters of the GLCT-SECIHURD model *
+***********************************************/
+
+/// @brief Vector with the probability to start in any of the subcompartments of the Exposed compartment.
+struct StartingProbabilitiesExposed {
+    using Type = Eigen::VectorXd;
+    /** 
+     * @brief Default parameters can be used to get an Erlang distributed stay time in the Exposed compartment.
+     * @param[in] numExposed Number of subcompartments of the Exposed compartment.
+     */
+    static Type get_default(size_t numExposed)
+    {
+        Eigen::VectorXd def = Eigen::VectorXd::Zero(numExposed);
+        def[0]              = 1.;
+        return def;
+    }
+    static std::string name()
+    {
+        return "StartingProbabilitiesExposed";
+    }
+};
+
+/// @brief Transition Matrix of the Exposed compartment.
+struct TransitionMatrixExposedToInfectedNoSymptoms {
+    using Type = Eigen::MatrixXd;
+    /**
+     * @brief Default parameters can be used to get an Erlang distributed stay time in the Exposed compartment.
+     * @param[in] numExposed Number of subcompartments of the Exposed compartment.
+     * @param[in] timeExposed Average time spent in Exposed compartment in day unit.
+     */
+    static Type get_default(size_t numExposed, ScalarType timeExposed = 2.)
```

Comment:
How do you choose the default values for the times? This changes throughout the compartments but I don't see why you chose them the way you did.